packages/botonic-react/tests/webchat.theme.test.js

```diff
@@ -94,3 +95,72 @@ describe('Updating webchat properties with settings ', () => {
     )
   })
 })
+
+describe('Deep merging theme properties', () => {
+  it('Preserve theme deep properties', () => {
+    const theme1 = {
+      theme: {
+        userInput: { box: { placeholder: 'placeholder...' } },
+        header: {
+          title: 'title',
+        },
+        persistentMenu: [
+          {
+            label: 'Start',
+            payload: 'start',
+          },
+          {
+            label: 'Help',
+            payload: 'help',
+          },
+          { closeLabel: 'close' },
+        ],
+      },
+    }
+    const theme2 = {
+      theme: {
+        userInput: {
+          style: {
+            background: 'white',
+            minHeight: '45px',
+          },
+          box: {
+            style: {
+              border: 'none',
+              paddingLeft: 20,
+              marginRight: 10,
+            },
+            placeholder: 'another placeholder...',
+          },
+          menu: {
+            darkBackground: true,
+          },
+        },
+      },
+    }
+    const result = {
+      theme: {
+        userInput: {
+          box: {
+            placeholder: 'another placeholder...',
+            style: {
+              border: 'none',
+              paddingLeft: 20,
+              marginRight: 10,
+            },
+          },
+          style: { background: 'white', minHeight: '45px' },
+          menu: { darkBackground: true },
+        },
+        header: { title: 'title' },
+        persistentMenu: [
+          { label: 'Start', payload: 'start' },
+          { label: 'Help', payload: 'help' },
+          { closeLabel: 'close' },
+        ],
+      },
+    }
+    const sut = merge(theme1, theme2)
```

Comment:
I see two typical issues in unit tests:
a) unit tests should test our code, not third party functions (which probably have their own unit tests). The only test where it makes sense, is when then 3rd party function is complex (and in this case it's better to wrap it and then test the wrapped function.
b) the unit test is very long and visually it's difficult to appreciate which fields are merged. 

I suggest to:
* leave this test reducing the theme to three fields tops
* adding an empty test.skip which  is called st like webchab_merges_theme_from_xx_with_theme_from_yy